update-po.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Why do we need this script and the `update-pot.sh` script, rather than simply using the `Intltool.cmake` functions in `po/CMakeLists.txt` instead?

Reply:
I was following the established model for all Ayatana Indicators using CMake:
https://github.com/AyatanaIndicators/ayatana-indicator-session/blob/master/update-po.shhttps://github.com/AyatanaIndicators/ayatana-indicator-datetime/blob/master/update-po.sh

Reply:
Huh. Can you answer this question of mine @sunweaver ?